cmd/jujud/agent/util_test.go

```diff
@@ -167,11 +167,29 @@ func (s *commonMachineSuite) primeAgent(c *gc.C, jobs ...state.MachineJob) (m *s
 	return s.primeAgentVersion(c, vers, jobs...)
 }
 
+// TODO(wallyworld) - we need the dqlite model database to be available.
```

Comment:
Delete this for now?

Reply:
I left it in because the place which calls it is also there with a TODO to make it super obvious the bits we need to enable so we don't need to rediscover what to do when we can have the dqlite model db available.